.cloudbuild/scripts/cmd/integration-tests/main.go

```diff
@@ -152,23 +155,27 @@ func innerMain() error {
 	return nil
 }
 
-func runRootIntegrationTests(workspace string, env ...string) error {
+func runRootIntegrationTests(workspace string, env map[string]string) error {
 	// Run root integration tests
 	cmd := exec.Command("make", "rdpclient", "integration-root")
 	cmd.Dir = workspace
-	if len(env) > 0 {
-		cmd.Env = append(os.Environ(), env...)
+	cmd.Env = os.Environ()
+	for k, v := range env {
+		cmd.Env = append(cmd.Env, fmt.Sprintf("%v=%v", k, v))
 	}
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 
 	return cmd.Run()
 }
 
-func runNonrootIntegrationTests(workspace string, uid, gid int, env ...string) error {
+func runNonrootIntegrationTests(workspace string, uid, gid int, env map[string]string) error {
 	cmd := exec.Command("make", "integration")
 	cmd.Dir = workspace
-	cmd.Env = append(append(os.Environ(), "TELEPORT_ETCD_TEST=yes"), env...)
+	cmd.Env = append(append(os.Environ(), "TELEPORT_ETCD_TEST=yes"))
```

Comment:
Shouldn't this be just one append?
```suggestion
	cmd.Env = append(os.Environ(), "TELEPORT_ETCD_TEST=yes")
```

Reply:
Fixed, thanks.